components/admin-tabs/Applicants.tsx

```diff
@@ -187,44 +198,51 @@ const Applicants: React.FC = () => {
 
 const escaper = (s: string) => `"${s.replaceAll('"', '""')}"`;
 const separator = ',';
-const fileName = 'applicants.csv';
-const downloadFile = async (totalCount: number, filters: TableFilters, sorter: TableSorter) => {
+const fields: Array<keyof User> = ['email', 'isAdmin', 'applicationStatus', 'rsvpStatus'];
+type DownloadProps = {
+  totalCount: number;
+  filters: TableFilters;
+  sorter: TableSorter;
+};
+const downloadApplicationCsv = async (props: DownloadProps) =>
+  downloadFileAbstract(props, fields, Questions, (u) => u.responses, 'applications.csv');
+const downloadPostAcceptanceCsv = async (props: DownloadProps) =>
+  downloadFileAbstract(
+    props,
+    fields,
+    PostAcceptanceFormQuestions,
+    (u) => u.postAcceptanceResponses,
+    'post-acceptance.csv'
+  );
+
+const serializeResponse = (response: QuestionResponse) => {
+  if (Array.isArray(response)) {
+    return response.join('\n');
+  }
+  return response ?? '';
+};
+const downloadFileAbstract = async (
+  { totalCount, filters, sorter }: DownloadProps,
+  fields: Array<keyof User>,
+  questions: QuestionDefinition[],
+  responseGetter: (u: User) => Array<QuestionResponse> | undefined,
```

Comment:
just pulled out some variables to abstract this function. `fields` is the fields of user to include (see line 201), `questions` is the question definitions to add columns for (either the post acceptance qs or the application qs), and `responseGetter` is a function to get the responses from the user (either grabs `postAcceptanceResponses` or `responses`.